lambdas/service/vendor/static/swagger-ui.html.template.mustache

```diff
@@ -43,11 +43,14 @@
         plugins: [
           SwaggerUIBundle.plugins.DownloadUrl
         ],
-        oauth2RedirectUrl: "{{OAUTH2_REDIRECT_URL}}"
-      })
-      ui.initOAuth({
-        clientId: "{{OAUTH2_CLIENT_ID}}"
+        oauth2RedirectUrl: "{{{OAUTH2_REDIRECT_URL}}}"
```

Comment:
Anyways, I don't think we need to split the commits, but 
```suggestion
        oauth2RedirectUrl: {{{OAUTH2_REDIRECT_URL}}}
```

and json.dumps above.